VMEncryption/main/oscrypto/OSEncryptionState.py

```diff
@@ -149,19 +149,19 @@ def _get_fs_partition(self, fs):
         result = None
         dev = os.lstat(fs).st_dev
 
-        for line in file('/proc/mounts'):
-            line = [s.decode('string_escape') for s in line.split()[:3]]
+        for line in open('/proc/mounts'):
+            line = [bytes(s,'utf-8').decode('unicode_escape') for s in line.split()[:3]]
```

Comment:
update prior to match unicode_escape syntax

Reply:
updated to use same mechanism from DiskUtil.py for parsing mount points -- but thinking this might be better moved into DiskUtil.py for all mount items and then sort through that result.